base/applications/cmdutils/fc/lang/de-DE.rc

```diff
@@ -2,40 +2,40 @@ LANGUAGE LANG_GERMAN, SUBLANG_NEUTRAL
 
 STRINGTABLE
 BEGIN
-    IDS_USAGE "Compares two files or sets of files and displays the differences between\n\
-them\n\
+    IDS_USAGE "Vergleicht zwei Dateien oder Gruppen von Dateien und zeigt die\n\
+Unterschiede zwischen ihnen an\n\
 \n\
 FC [/A] [/C] [/L] [/LBn] [/N] [/OFF[LINE]] [/T] [/U] [/W] [/nnnn]\n\
-   [drive1:][path1]filename1 [drive2:][path2]filename2\n\
-FC /B [drive1:][path1]filename1 [drive2:][path2]filename2\n\
+   [laufwerk1:][pfad1]dateiname1 [laufwerk2:][pfad2]dateiname2\n\
+FC /B [laufwerk1:][pfad1]dateiname1 [laufwerk2:][pfad2]dateiname2\n\
```

Comment:
Start nouns with capital letters, like german requests and MS does as well:
![image](https://user-images.githubusercontent.com/33393466/194292403-8b4d7930-ca49-4a99-abd8-7b23b85e5fe1.png)